### DISCUSSION TITLE
[Feature Request] LaTeX textbox support

### DISCUSSION THREAD

**Discussion by liar666** [SVG-Edit/svgedit#803] [New feature request]
Hi,

With the current Health crisis, many teachers are looking for online/shared whiteboards, and particularly ones that would support LaTeX equations (just search for "latex whiteboard" on your preferred SE :) ).

My maths & physics colleagues are also. As a proponent of FOSS, I was searching for a solution that we could install on a server in my university so that our work is not golden-jailed in any proprietary web tool. Unfortunately, I could not find one that would be both complete (i.e. as both whiteboard & LaTeX possibilities) & free (not as in beer).

That's when I remembered one could include LaTeX in Xfig and thus probably in Scribus too. Also, I remembered there was a nice online Scribus-like tool, yours :).

Unfortunately, it doesn't seem to support LaTeX out of the box :[.

However, during my surfing, looking for a solution for my colleagues, I found the following tool:
http://mathdox.org/formulaeditor/

I have no idea if it is doable, but it would be very nice if it could be included in SVG-Edit!!
Or maybe if it is too complicated simply having a way to render a LaTeX expression inside a textbox would be niiiiiiiice!

Thanks for your work!

**Comment by brettz9** [comment]
Though I know LaTeX is more broad than TeX, you might take a look at the Mathjax extension (added to the UI by adding `?extensions=ext-mathjax.js` to the SVGEdit URL. This has some TeX capabilities, though seems it may not be working. We badly need some tests. (I have a "mathjax" branch started to try to get things working with the latest Mathjax version, but haven't had time to get it working.)

Anyways, well documented PRs with UI tests are welcome, but this project is not being actively maintained, with the original developers no longer active.
